server/index.js

```diff
@@ -89,17 +89,17 @@ const session = async (app) => {
       req.currentUser = await app.objection.models.user.query().findById(userId);
     }
   });
-  app.addHook('preHandler', async (req) => {
-    console.log('============');
-    console.log(`REQ - ${req.method} ${req.url} ${JSON.stringify(req.body)}`);
-    console.log('------------');
-  });
-  app.addHook('onSend', async (req, reply, payload) => {
-    if (!req.url.includes('assets')) {
-      console.log(`RES - ${req.method} ${req.url} ${reply.statusCode}\n${payload}`);
-      console.log('============');
-    }
-  });
+  // app.addHook('preHandler', async (req) => {
+  //   console.log('============');
+  //   console.log(`REQ - ${req.method} ${req.url} ${JSON.stringify(req.body)}`);
+  //   console.log('------------');
+  // });
+  // app.addHook('onSend', async (req, reply, payload) => {
+  //   if (!req.url.includes('assets')) {
+  //     console.log(`RES - ${req.method} ${req.url} ${reply.statusCode}\n${payload}`);
+  //     console.log('============');
+  //   }
+  // });
```

Comment:
it's common not to add debug code to git

Reply:
removed